presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientConfig.java

```diff
@@ -151,4 +155,18 @@ public boolean isGroupedExecutionEnabled()
     {
         return groupedExecutionEnabled;
     }
+
+    @NotNull
+    public Duration getDynamicFilteringWaitTimeout()
+    {
+        return dynamicFilteringWaitTimeout;
+    }
+
+    @Config("kudu.dynamic-filtering-wait-timeout")
+    @ConfigDescription("Duration to wait for completion of dynamic filters")
+    public KuduClientConfig setDynamicFilteringWaitTimeout(Duration dynamicFilteringWaitTimeout)
```

Comment:
Let's use `@MinDuration("0ms")` annotation

Reply:
Added @MinDuration("0ms") on the getter as other config